WooCommerce/src/main/kotlin/com/woocommerce/android/ui/mystore/MyStoreStatsView.kt

```diff
@@ -93,45 +85,41 @@ class MyStoreStatsView @JvmOverloads constructor(
 
     private val fadeHandler = Handler(Looper.getMainLooper())
 
-    private val visitorsLayout
-        get() = binding.root.findViewById<ViewGroup>(R.id.visitors_layout)
-
-    private val visitorsValue
-        get() = binding.root.findViewById<MaterialTextView>(R.id.visitors_value)
+    private val statsDateValue
+        get() = binding.root.findViewById<MaterialTextView>(R.id.statsDateTextView)
```

Comment:
I believe we can simplify this expression to this:
```suggestion
        get() = binding.statsViewRow.statsDateTextView
```

Same applies to the properties below.